django/core/management/commands/sendtestemail.py

```diff
@@ -1,21 +1,35 @@
 import datetime
 import socket
 
+from django.conf import settings
 from django.core.mail import send_mail
 from django.core.management.base import BaseCommand
 
 
 class Command(BaseCommand):
     help = "Sends a test email to the email addresses specified as arguments."
+    missing_args_message = "You must specify some email recipients, or pass the --managers or --admin arguments."
```

Comment:
Have changed arguments to options.

Could you clarify your other point - are you suggesting we may not need it at all, or that it needs a test?